src/picongpu/include/particles/ionization/ionizationMethods.hpp

```diff
@@ -71,7 +71,7 @@ namespace ionization
              * - boundElectrons: because species other than ions or atoms do not have them
              * (gets AUTOMATICALLY deselected because electrons do not have this attribute)
              */
-            PMACC_AUTO(targetElectronClone, partOp::deselect<bmpl::vector2<multiMask, momentum> >(childElectron));
+            PMACC_AUTO(targetElectronClone, partOp::deselect<bmpl::vector2<multiMask, momentum, particleId> >(childElectron));
```

Comment:
old bug: multiMask is now uninitialized in the new species
new bug: deselect particleId in assign